domains/chankit.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ChankitSaini",
+        "email": "chankit@outlook.in"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=chankit.is-a.dev,d7420a0e3e7289118bff"
```

Comment:
We do not support Vercel.

Reply:
> We do not support Vercel.

updated the records with cloudflare pages CNAME